### PR TITLE
adds vacuum part detection page

### PR DESCRIPTION
**Pull request by sphawes**
hey @gabeseltzer!

i made a page about configuring and enabling the vacuum sensors for both v2 and v3 machines. if you could take a peek and merge this in, that would be stellar. thank you sir!

### CODE REVIEW

File: docs/openpnp/calibration/10-vacuum-sensor/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+
+# Vacuum Part Detection
+
+The bottom camera can do a great job of detecting if a part has been picked successfully. The LumenPnP comes with integrated vacuum sensors that can add an extra layer of verification to pick success. When the LumenPnP picks a part, if it detects a vacuum in the pneumatic line, it knows a component is making a seal with the nozzle tip and we've picked a part.
```
Comment (theacodes) [inline docs/openpnp/calibration/10-vacuum-sensor/index.md:4]:
```suggestion
While the bottom camera can detect if a part was successfully picked with reasonable accuracy, you can also use the LumenPnP's vacuum sensors to double check a successful pick. The way this works is that when a part is successfully picked, it creates a seal at the end of the nozzle. This seal increases the vacuum pressure in the pneumatic line which is picked up by the vacuum sensors. OpenPnP can use this increase in pressure to detect if a part was successfully picked.

```

File: docs/openpnp/calibration/10-vacuum-sensor/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+
+# Vacuum Part Detection
+
+The bottom camera can do a great job of detecting if a part has been picked successfully. The LumenPnP comes with integrated vacuum sensors that can add an extra layer of verification to pick success. When the LumenPnP picks a part, if it detects a vacuum in the pneumatic line, it knows a component is making a seal with the nozzle tip and we've picked a part. 
+We switched vacuum sensors between v2 and v3 versions of the machine. Make sure you're using the correct config for your hardware.
```
Comment (theacodes) [inline docs/openpnp/calibration/10-vacuum-sensor/index.md:5]: I'd move this under the Actuator setup heading.

```suggestion
Note that LumenPnP v2 has different vacuum sensors than v3, so the setup is a little different. Make sure to follow the steps for your machine version below.
```

**Review by theacodes — CHANGES_REQUESTED** [review]

File: docs/openpnp/calibration/10-vacuum-sensor/index.md (hunk 2 above)
Comment (gabeseltzer) [reply docs/openpnp/calibration/10-vacuum-sensor/index.md:6]: I agree, and I'll move it to the actuator setup heading

**Review by gabeseltzer — APPROVED** [review]
thanks for writing and reviewing!

**Pull request merged**